recipes/myproxyclient/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "MyProxyClient" %}
+{% set version = "1.4.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "5b7ad3cc3c3cd71595a9b0278a51c138699e229d47a41f2e2add72fbc12f58cd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  entry_points:
+    - myproxyclient = myproxy.script:main
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyopenssl
+    - certifi
+
+  run:
+    - python
+    - pyopenssl
+
+test:
+  imports:
+    - myproxy
+    - myproxy.test
+    - myproxy.utils
+  commands:
+    - myproxyclient --help
+
+about:
+  home: https://github.com/cedadev/MyProxyClient
```

Comment:
Despite originally being a fork, it appears to be the main repo being used as the main repo according to the maintainers as implied [here](https://github.com/philipkershaw/MyProxyClient/issues/3). The previous release version showed a different homepage (http://ndg-security.ceda.ac.uk/wiki/MyProxyClient) but it doesn't appear to be currently maintained.